__tests__/new/swapConnection.test.ts

```diff
@@ -0,0 +1,108 @@
+import { AtomicSwapServiceConnection } from '../../src/swapService/swapConnection';
```

Comment:
The folder of the test should change as well?

Reply:
Fixed on [b6207ef](https://github.com/HathorNetwork/hathor-wallet-lib/pull/533/commits/b6207efa1e23251ee6b77e9fc0039459d6347594)